### ISSUE TITLE
Error: Command failed with ENOENT: /usr/local/bin/brew install make

### ISSUE THREAD

**Issue by lczech** [aminya/setup-cpp#247]
The following GitHub Actions workflow fails under `os: macos-latest`

```
uses: aminya/setup-cpp@v0.37.0
with:
  compiler: ${{ matrix.compiler }}
  cmake: true
  make: true
```

with the error `Error: Command failed with ENOENT: /usr/local/bin/brew install make`

In more detail:

```
Run aminya/setup-cpp@v0.37.0
  with:
    compiler: gcc
    cmake: true
    make: true
    timeout: 20
Installing cmake true
  Installing cmake 3.28.1 arm64 via direct downloading
  Download cmake 3.28.1
  Trying https://github.com/Kitware/CMake/releases/download/v3.28.1/cmake-3.28.1-macos-universal.tar.gz #1 at 3:37:19 PM
  Installing extraction dependencies
  Extracting /Users/runner/work/_temp/5fd0404b-e7a3-4250-988a-41452e78260e to /Users/runner/cmake
  /usr/bin/tar xz -C /Users/runner/cmake -f /Users/runner/work/_temp/5fd0404b-e7a3-4250-988a-41452e78260e
  Add /Users/runner/cmake/cmake-3.28.1-macos-universal/CMake.app/Contents/bin/ to PATH
took 5 seconds
Installing make true
  Installing make  via brew
  Error: Error: Command failed with ENOENT: /usr/local/bin/brew install make
  spawnSync /usr/local/bin/brew ENOENT
took 0 seconds
✅ cmake was installed successfully:
- The installation directory is /Users/runner/cmake/cmake-3.28.1-macos-universal
- The binary directory is /Users/runner/cmake/cmake-3.28.1-macos-universal/CMake.app/Contents/bin/
Error: make failed to install
setup-cpp finished
```

This could be an issue with npm, see [here](https://stackoverflow.com/a/17324698/4184258)

Cheers and thanks in advance for any hints on how to solve this!
Lucas

## Upvote & Fund

@aminya is using [Polar.sh](https://polar.sh/aminya) so you can upvote and help fund this issue. The funding is received once the issue is completed & confirmed by you.

Thank you in advance for helping prioritize & fund our backlog!

<br/>

<a href="https://polar.sh/aminya/setup-cpp/issues/247">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/aminya/setup-cpp/issues/247/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/aminya/setup-cpp/issues/247/pledge.svg">
</picture>
</a>

**Comment by aminya** [comment]
There's a check to determine if brew is installed. Not sure why this error is being triggered.

https://github.com/aminya/setup-cpp/blob/34bb7838132719f0db8a21d1b1c805d39f21977d/src/utils/setup/setupBrewPack.ts#L19

https://github.com/aminya/setup-cpp/blob/34bb7838132719f0db8a21d1b1c805d39f21977d/src/make/make.ts#L18

**Comment by aminya** [comment]
Fixed in v0.41.0. Now, Arm MacOS is fully tested and supported.
